shared/src/main/scala/amf/core/internal/remote/InternalContent.scala

```diff
@@ -0,0 +1,19 @@
+package amf.core.internal.remote
+
+import amf.core.client.common.remote.Content
+import amf.core.client.scala.lexer.CharStream
+
+/**
+  *
+  */
+case class InternalContent(stream: CharStream, url: String, mime: Option[String] = None)
+
+object InternalContent {
```

Comment:
"Rough" name. How can I tell what the difference is between an `InternalContent` and a `Content` ? Functionally speaking.